x-pack/plugins/security_solution/common/utils/path_placeholder.ts

```diff
@@ -16,7 +16,7 @@ export const placeholderText = {
     wildcard: '/opt/**/*',
     exact: '/opt/bin',
   },
-};
+});
 
 export const getPlaceholderText = ({
```

Comment:
And this could be changed by: `getPlaceholderTextByFieldType()` or something similar as this is returning a placeholderText depending on the field type.

Reply:
done ab49010e404be6dc2955102a7608ec2bb29a5413